### DISCUSSION TITLE
Create ann app in Android

### DISCUSSION THREAD

**Discussion by puentejo** [community/community#129457] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Can you point me in the right direction of a tutorial that can help me in how to create an app for Android ?

**Comment by davevad93** [comment]
Hi @puentejo ,

First of all, the best place to start is the official Android documentation: 

- [Create your first Android app](https://developer.android.com/codelabs/basic-android-kotlin-compose-first-app#0)

Also, there are a lot of good tutorials about `Android` app development, like this `YouTube` video from `freeCodeCamp`:

- [Android App Development Tutorial for Beginners - Your First App](https://www.youtube.com/watch?v=FjrKMcnKahY)

If you want to take a series of free courses to get you started with `Android`, I recommend this learning path from its official site:

- [Android Basics with Compose](https://developer.android.com/courses/android-basics-compose/course)

Hope it helps!

**Comment by nandodev-net** [comment]
Sure!, @puentejo, here are some great resources to help you get started with creating an Android app:

freeCodeCamp's Android App Development for Beginners: This tutorial covers the basics of Android app development, from setting up your environment to creating a simple, functional app. It is an excellent resource for beginners. You can watch it [here on YouTube](https://www.youtube.com/watch?v=fis26HvvDII)​ ([FreeCodeCamp](https://www.freecodecamp.org/news/android-app-development-for-beginners/))​.

Android Developers' "Build your first app": This is an official tutorial from Google that guides you through building your first Android app using Android Studio and Kotlin. It covers the essentials of app development and provides hands-on coding experience. Start the course [here](https://developer.android.com/training/basics/firstapp)​ ([Android Developers](https://developer.android.com/get-started/overview))​​ ([Android Developers](https://developer.android.com/courses/android-basics-compose/course))​.

Android Basics with Compose: This course is part of Google's official training materials and focuses on building Android apps using Jetpack Compose, a modern toolkit for developing user interfaces. The course is self-paced and covers fundamental topics in Kotlin and Android app development. You can begin this course [here](https://developer.android.com/courses/android-basics-compose/course)​ ([Android Developers](https://developer.android.com/courses/android-basics-compose/course))​.

These resources should provide a solid foundation for developing Android apps, from setting up your development environment to building and deploying your first app. Good luck!

**Comment by ebndev** [comment]
Hi @puentejo, thanks for being a part of the GitHub Community!

Looks like you've got some great responses above from @davevad93 & @nandodev-net. Please feel free to follow-up with them if you have questions on their comments 😄

**Comment by Sworzen1** [comment]
Hello @puentejo ,
I would like to encourage you to take advantage of it and advertise the [Expo](https://expo.dev/) at the same time.
Its really simple and usefull tool for developers if you know JavaScript you can create and maintain Android, iOS and Web applications.
Take a moment and read about Expo :)
